DBpia.js

```diff
@@ -0,0 +1,186 @@
+	} else if ((url.indexOf('/SearchResult/')>-1 || url.indexOf('/Journal/ArticleList/')>-1) && getSearchResults(doc, true)) {
+		return "multiple";
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//div[@class="titleWarp"]/a');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var c = i + 1;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[c] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var num = [];
+			for (var i in items) {
+				num.push(i);
+			}
+			for (var i=0; i<num.length; i++) {
+				var n = num[i];
+				scrapeMulti(doc, url, n);
+			}
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrapeMulti(doc, url, n) {
+	var link = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dt/div/a/@href')
+	if (link.indexOf('book')==-1) {
+		ZU.processDocuments(link, scrape);
+	} else {
+		var item = new Zotero.Item("book");
+		var title = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dt/div/a');
+		item.title = title;
+		var creators = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dd[1]');
+		if (creators) {
+			var creatorsList = creators.split(',');
+			for (var i=0; i<creatorsList.length; i++) {
+				item.creators.push(ZU.cleanAuthor(creatorsList[i], "author", true));
+			}
+		}
+		item.publisher = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dd[2]/a');
+		item.date = ZU.xpathText(doc, '//div[@class="subSearchResultList"]/div[' + n + ']/ul[1]/li[4]/dl/dd[3]');
+		item.complete();
+	}
+}
+
+function scrape(doc, url) {
+	var item = new Zotero.Item("journalArticle");
+	var title = ZU.xpathText(doc, '//meta[@name="citation_title"]/@content');
+	item.title = title;
+	
+	var creators = ZU.xpathText(doc, '//meta[@name="citation_author"]/@content');
+	if (creators) {
+		var creatorsList = creators.split(',');
+		for (var i=0; i<creatorsList.length; i++) {
+			item.creators.push(ZU.cleanAuthor(creatorsList[i], "author", true));
+		}
+	}
+	
+	item.date = ZU.xpathText(doc, '//meta[@name="citation_publication_date"]/@content');
+	item.publicationTitle = ZU.xpathText(doc, '//meta[@name="citation_journal_title"]/@content');
+
+	var pagerange = ZU.xpathText(doc, '//div[@class="book_info"]/dl/dt[1]').match(/\s([\d+-\d+])\s/);
+		if (pagerange) {
+			item.pages = pagerange[1];
+		}
+		
+	item.volume = ZU.xpathText(doc, '//meta[@name="citation_volume"]/@content');
+	item.issue = ZU.xpathText(doc, '//meta[@name="citation_issue"]/@content');
+	item.ISSN = ZU.xpathText(doc, '//meta[@name="citation_issn"]/@content');
```

Comment:
This looks like the Embedded Metadata translator should already give you pretty good data. I would suggest to first call EM translator and then add the missing fields only.